### ISSUE TITLE
One Junction System

### ISSUE THREAD

**Issue by SimonRubenDrauz** [e2nIEE/pandapipes#145]
If you create just a one node system without any pipes, the solver can't find a solution and won't converge. This shouldn't be the case!

**Comment by dlohmeier** [comment]
We need to perform a check here and definitely, we have to continue throwing out equations that we don't need in the pipeflow.

**Comment by jkisse** [comment]
```
net2 = pp.create_empty_network('test', 'hgas')
pp.create_junction(net2, pn_bar=1, tfluid_k=290)
pp.create_sink(net2, 0, 0.1)
pp.create_ext_grid(net2, 0, p_bar=1, t_k=290)
pp.pipeflow(net2)

```
leads to 
```
hp.pandapipes.pipeflow - INFO: niter 0
C:\Users\...\pandapipes_jkisse\pandapipes\pandapipes\pipeflow.py:136: RuntimeWarning:
invalid value encountered in double_scalars
hp.pandapipes.pipeflow - INFO: niter 1
hp.pandapipes.pipeflow - INFO: niter 2
hp.pandapipes.pipeflow - INFO: niter 3
hp.pandapipes.pipeflow - INFO: niter 4
hp.pandapipes.pipeflow - INFO: niter 5
hp.pandapipes.pipeflow - INFO: niter 6
hp.pandapipes.pipeflow - INFO: niter 7
hp.pandapipes.pipeflow - INFO: niter 8
hp.pandapipes.pipeflow - INFO: niter 9
hp.pandapipes.pipeflow - INFO: niter 10
hp.pandapipes.pipeflow - INFO: ---------------------------------------------------------------------------------
hp.pandapipes.pipeflow - WARNING: Maximum number of iterations reached but hydraulics solver did not converge.
hp.pandapipes.pipeflow - INFO: Norm of residual: 0.0
```

--> Check how Norm of residual is handled and whether there are NaNs which may lead to "non-convergence"

**Comment by jkisse** [comment]
It seems there might be a related issue in pandapower: https://github.com/e2nIEE/pandapower/issues/780

**Comment by SimonRubenDrauz** [comment]
Should be solved in PR #159. When verified, Issue can be closed!
